A <%= exception.class.to_s %> occured: <%= exception.message.to_s %>
<% if exception.respond_to?(:backtrace)  %>
<% if exception.backtrace.present?  %>
  ===================================================================
  Backtrace:
  ===================================================================

  <%= exception.backtrace.slice(0, 5).join("\n  ") %>
<% end %>
<% end %>
  ===================================================================
Rack Environment:
===================================================================

PID:                     <%= process %>
PWD:                     <%= pwd %>
Uname:                   <%= uname %>
Server:                  <%= server %>
Error time:	             <%= timestamp %>
Request URI:             <%= request_path %>
Referer:	               <%= referrer %>
User Agent:	               <%= user_agent %>
